src/routes.js

```diff
@@ -4,6 +4,7 @@ routes
   .add("home", "/", "index")
   .add("cart", "/cart", "cart")
   .add("checkout", "/cart/checkout", "checkout")
+  .add("checkout-complete", "/checkout/order/:orderId", "checkoutComplete")
```

Comment:
Looks like shopProduct is camelCase so change `checkout-complete` to `checkoutComplete`